extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -256,23 +300,7 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     const width = Number(this.element.getAttribute('width'));
     const height = Number(this.element.getAttribute('height'));
 
-    const adsenseFormatExpName = 'as-use-attr-for-format';
-    const experimentInfoMap =
-        /** @type {!Object<string,
-        !../../../src/experiments.ExperimentInfo>} */ ({});
-    experimentInfoMap[adsenseFormatExpName] = {
-      isTrafficEligible: () => !this.isResponsive_() &&
-        !isNaN(width) && width > 0 &&
-        !isNaN(height) && height > 0,
-      branches: ['21062003', '21062004'],
-    };
-
-    const adsenseFormatExpId =
-        randomlySelectUnsetExperiments(
-            this.win, experimentInfoMap)[adsenseFormatExpName];
-    addExperimentIdToElement(adsenseFormatExpId, this.element);
-
-    this.size_ = adsenseFormatExpId == '21062004'
+    this.size_ = getExperimentBranch(this.win, FORMAT_EXP) == '21062004'
```

Comment:
I realize you didn't name this attribute in this PR, but this seems like a bad name if it is a bool....